src/Files/Views/LayoutModes/DetailsLayoutBrowser.xaml

```diff
@@ -633,9 +634,27 @@
                                         TextTrimming="CharacterEllipsis" />
                                     <TextBox
                                         x:Name="ItemNameTextBox"
+                                        MinWidth="200"
+                                        HorizontalAlignment="Left"
                                         VerticalAlignment="Center"
                                         TextChanged="ListViewTextBoxItemName_TextChanged"
-                                        Visibility="Collapsed" />
+                                        Visibility="Collapsed">
+                                        <TextBox.Resources>
```

Comment:
Isn't this automatic?

Reply:
@yaichenbaum This resource makes the background opaque to prevent the columns behind it from showing.